utils/rpms/packaging/Dockerfile.mockbuild

```
@@ -1,12 +1,12 @@
 #
-# Copyright 2018-2020, Intel Corporation
+# Copyright 2018-2021, Intel Corporation
 #
 # 'recipe' for Docker to build an RPM
 #
 
 # Pull base image
 FROM fedora:latest
-MAINTAINER daos-stack <daos@daos.groups.io>
+LABEL maintainer="daos@daos.groups.io>"
```

Did we miss removing the `>`?
 ```suggestion
LABEL maintainer="daos@daos.groups.io"
```

---

Fixed in next push